src/issues/issueCompletionProvider.ts

```diff
@@ -182,7 +182,7 @@ export class IssueCompletionProvider implements vscode.CompletionItemProvider {
 			const configuration = vscode.workspace
 				.getConfiguration(ISSUES_CONFIGURATION)
 				.get('issueCompletionFormatScm');
-			if (document.uri.path.match(/scm\/git\/scm\d\/input/) && typeof configuration === 'string') {
+			if (document.uri.path.match(/git\/scm\d\/input/) && typeof configuration === 'string') {
```

Comment:
Would it make sense to instead check that:
1. document URI schema is `vscode`
2. document URI authority is `scm`